NSubstitute.DbConnection/NSubstitute.DbConnection/QueryParameter.cs

```diff
@@ -8,17 +10,17 @@ public QueryParameter(string name, object value)
             Value = value;
         }
 
-        public QueryParameter(string name, object value, bool isOutput)
+        public QueryParameter(string name, object value, ParameterDirection direction)
         {
             Name = name;
             Value = value;
-            IsOutput = isOutput;
+            Direction = direction;
         }
 
         public string Name { get; set; }
 
         public object Value { get; set; }
 
-        public bool IsOutput { get; set; }
+        public ParameterDirection Direction { get; set; } = ParameterDirection.Input;
```

Comment:
Just noting that since this is internal, it's okay that the interface is changing here